### ISSUE TITLE
A11y_SqlDatabaseProjectsExtension_CreateNewDatabaseProject_SQLServerdatabase_ScreenReader: ORCA Screen reader is not announcing "selected" state of list item from the "Target platform" combobox.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#23559]
### Actual Result:
ORCA Screen reader is not announcing "selected" state of list item from the "Target platform" combo box.

### Expected Result:
ORCA Screen reader should announce "selected" state of list item from the "Target platform" combo box.

### Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #Benchmark; #ORCA; #WCAG1.3.1; #FTP; #SH-SQL Azure Data Studio-Linux-Jul23; #SQL Azure Data Studio; #Linux; #Element:Combobox;

### Environment Details:
Application Name: Sql Database Projects Extension
Operating System: Ubuntu 20.04.2 LTS
Screen Reader: ORCA

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to 'Extension' and search for “SQL Database Projects” and install it.
3. TAB to "Database project" and press ENTER key.
4. Start ORCA screen reader. TAB till “Create New” button and press ENTER key.
5. “Create New Database Project” window will open. TAB till type section and tab till SQL Server Database radio button to select it.
6. TAB to "Target platform" combo box and hit ENTER key.

### User Impact:
The screen reader users will get incorrect information and will get confused whether the list item is selected or not 

### Recommendations:
Refer [Repository of bug fixes code snippets](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2faka.ms%2faccsnippet&data=05%7c01%7cv-samhithaks%40microsoft.com%7c535b4a834ef843acdfb508da53769f91%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637914066217264155%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=xoIfLB9Iq6HVIPyUc42rS%2BlVQCt9m%2BaUokp32Kaz1nE%3D&reserved=0).

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

**Comment by kisantia** [comment]
Similar issue as https://github.com/microsoft/azuredatastudio/issues/23182, which was closed. 

@PoojaNamde this seems to be an issue with ORCA, we are just using a standard select element here, you can reproduce the same issue with this example here: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:08-16-23;

Closing this bug as per the discussion with SME.
